articles/quickstarts/index.md

```diff
@@ -19,17 +19,144 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
 - APIs for Python and .NET languages (C#, F#, and VB.NET) for running quantum programs written in Q#
 - Tools to facilitate your development
 
-Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, or through Jupyter Notebooks with the IQ# Jupyter kernel.
-They can also be paired with a host program written in a .NET language (typically C#) or Python, enabling you to call quantum operations from inside a classical program.
+Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, through Jupyter Notebooks with the IQ# Jupyter kernel, or paired with a host program written in a .NET language (typically C#) or Python. You can also run Q# programs online using [Codespaces](https://online.visualstudio.com/) or [MyBinder.org](https://mybinder.org/). 
 
-The workflows for each of these setups are described and compared at [Ways to run a Q# program](xref:microsoft.quantum.guide.host-programs).
+## Installation options
```

Comment:
Meta-comment on this page - what is the reason to move all of the installation instructions for all of the platforms onto a single page? This is a very long page where most of the information will not be relevant (a given user is only looking for one particular platform, not all of them).

Did you consider perhaps having this page simply be the matrix of options, which contains links to each individual installation page for more details?

Reply:
The main reason is to have all options for local, online and docker in one page that we can share.
I'm fine with having the specific install instructions in separate pages, but in this case, let's also have the relevant getting started example in that same page.

Reply:
Yes, that's what I'm suggesting: leave the install instructions and getting started instructions in their own pages (as they already are today), and have this change simply add a landing page that makes all of the options visible in one place. I'm very concerned that separating the installation instructions from the getting started examples makes things more confusing and would cancel out the benefit of having this single landing page.